packages/wonder-blocks-data/util/response-cache.test.js

```diff
@@ -233,17 +714,33 @@ describe("./response-cache.js", () => {
             };
 
             // Act
-            const result = cache.getEntry(fakeHandler, "options");
+            const result = cache.removeAll(
+                fakeHandler,
+                (k, d) => d.data === "2",
+            );
+            const after = cache.cloneCachedData();
 
             // Assert
-            expect(result).toBeNull();
+            expect(result).toBe(2);
+            expect(after).toStrictEqual({
+                MY_HANDLER: {
+                    MY_KEY2: {data: "1"},
+                },
+                OTHER_HANDLER: {
+                    MY_KEY: {data: "1"},
+                },
+            });
```

Comment:
I sometimes struggle a bit reviewing tests where everything's mocked out (my fault not yours), so it's nice to see some tests with data as well. 🙂